File: backend/grafiosch-base/src/main/java/grafiosch/entities/Auditable_.java
package grafiosch.entities;

import java.time.LocalDateTime;

import jakarta.annotation.Generated;
import jakarta.persistence.metamodel.SingularAttribute;
import jakarta.persistence.metamodel.StaticMetamodel;

@Generated(value = "org.hibernate.jpamodelgen.JPAMetaModelEntityProcessor")
@StaticMetamodel(Auditable.class)
public abstract class Auditable_ {

  public static volatile SingularAttribute<Auditable, LocalDateTime> lastModifiedTime;
  public static volatile SingularAttribute<Auditable, LocalDateTime> creationTime;
  public static volatile SingularAttribute<Auditable, Integer> createdBy;
  public static volatile SingularAttribute<Auditable, Integer> lastModifiedBy;
  public static volatile SingularAttribute<Auditable, Integer> version;

}

File: frontend/src/app/lib/entities/auditable.ts
import {ProposeTransientTransfer} from './propose.transient.transfer';

export abstract class Auditable extends ProposeTransientTransfer {
  createdBy: number = null;
  creationTime: string = null;
  lastModifiedBy: number = null;
  lastModifiedTime: string = null;
  version: number = null;
}
